src/main/resources/CodyBundle.properties

```diff
@@ -185,6 +185,7 @@ filter.sidebar-panel-ignored-file.text=<html>This file has been restricted by an
 filter.sidebar-panel-ignored-file.learn-more-cta=Learn more
 
 # Other Actions
+action.cody.not-working=Cody is disabled or still starting up
```

Comment:
We already have messages for `notification.general.cody-not-running.title` and `notification.general.cody-not-running.detail`. Does this new message have a different use case?